src/sql/base/browser/ui/selectBox/media/selectBox.css

```diff
@@ -16,5 +16,5 @@
 
 .monaco-select-box {
 	padding: 2px 8px;
-	padding: 0 22px 0 6px !important; /* I don't like this but for now its fine */
+	padding: 0 22px 0 6px;
```

Comment:
Why are you removing the !important? I assume it was put there for a reason - did you make this doing this isn't breaking behavior anywhere else?

Reply:
I corrected the original style with a more specific selector -- that way it does not adversely impact other select boxes.